llvm/include/llvm/IR/IntrinsicsAMDGPU.td

```diff
@@ -150,6 +150,8 @@ defm int_amdgcn_workitem_id : AMDGPUReadPreloadRegisterIntrinsic_xyz;
 defm int_amdgcn_workgroup_id : AMDGPUReadPreloadRegisterIntrinsic_xyz_named
                                <"__builtin_amdgcn_workgroup_id">;
 
+defm int_amdgcn_grid_size : AMDGPUReadPreloadRegisterIntrinsic_xyz;
```

Comment:
This is dead code

Reply:
It's not dead, but that's unimportant - the patch could be reworked to not include it and I'll do that in the spirit of increasing consensus